iotqatools/iot_logger.py

```diff
@@ -25,12 +25,14 @@
 __author__ = 'xvc'
 
 import logging
+from iotqatools.helpers_utils import LogLevelConfiguration
 
 
-def get_logger(name, level='DEBUG', verbose=False, file=False, filename='', formatter=None):
+def get_logger(name, level=None, verbose=False, file=False, filename='', formatter=None):
     """
     :param name: Name of the logging module
-    :param level: Verbosity level (default DEBUG). Possible values ['CRITICAL','ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET')
+    :param level: Verbosity level (default None). Possible values [None, 'CRITICAL','ERROR', 'WARNING', 'INFO', 'DEBUG'].
```

Comment:
NOTSET was not being actually used and None will do the same with more semantic correctness from my point of view. Thus, I'm removing references to NOTSET in this PR.